_episodes/07-plot.md

```diff
@@ -239,13 +237,33 @@ As a simple example, notice that Matplotlib puts ticks on the outside
 of the figures by default, and only on the left and bottom sides of
 the axes.
 
-To change this behavior, you can use `gca()` to get the current axes
-and `tick_params` to change the settings.
+So far, everything we have wanted to do we could call directly from 
+the pyplot module with `plt.`. As you do more and more customization
+you may need to run some methods on plotting objects themselves. To use the 
+method that changes the direction of the ticks we need an `axes` object.
+So far, Matplotlib has implicitly created our `axes` object on the fly when we called `plt.plot`.
+To explicitly create an `axes` object we can first create our `figure` object and then add an `axes` object
+to it.
+~~~
+fig = plt.figure(figsize=(10,2.5))
+ax = fig.add_subplot(1,1,1)
+~~~
+{.language python}
+> Confusingly, in Matplotlib the objects `subplot` and `axes` are often be used interchangeably. 
+> This is because a `subplot` is an `axes` object with additional methods and attributes. 
```

Comment:
I'm not sure if this is true. subplot is a Figure object that returns a Axes object.  Do you mean to say, 

This is because `subplot` generates an `axes` object...

Reply:
or subplot is a Figure method? Regardless, updated to a more accurate portrayal.

Reply:
I wrote this based on  the fact that add_subplot returns a [axes.SubplotBase](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.SubplotBase.html#matplotlib.axes.SubplotBase) object which says "Base class for subplots, which are Axes instances with additional methods to facilitate generating and manipulating a set of Axes within a figure."